website/babel.config.js

```diff
@@ -25,7 +25,9 @@ const isMacrosName = packageName =>
   defaultMacrosRegex.test(packageName) &&
```

Comment:
It's a discussion in gatsby if the allow babel.config (which are more project config) or rc files. They said they want to support babel.config.js so I think it's okay and better to use it.